components/contributors/IndividualContributor.js

```diff
@@ -122,24 +122,15 @@ const IndividualContributor = ({ props }) => {
 }
 
 IndividualContributor.propTypes = {
-  props: PropTypes.shape({}).isRequired,
-  name: PropTypes.string,
-  login: PropTypes.string,
-  twitter_username: PropTypes.string,
-  avatar_url: PropTypes.string,
-  blog: PropTypes.string,
-  html_url: PropTypes.string,
-  contributions: PropTypes.string,
-}
-
-IndividualContributor.defaultProps = {
-  name: '',
-  login: '',
-  twitter_username: '',
-  avatar_url: '',
-  blog: '',
-  html_url: '',
-  contributions: '',
+  contributor: PropTypes.shape({
```

Comment:
The original list is all the properties in the object, but we never use them all, so refining the prop types for a contributor to the properties we render.